apps/editing-toolkit/package.json

```diff
@@ -94,6 +94,7 @@
 		"@automattic/composite-checkout": "*",
 		"@automattic/data-stores": "^1.0.0-alpha.1",
 		"@automattic/domain-picker": "^1.0.0-alpha.0",
+		"@automattic/page-template-modal": "^1.0.0-alpha.0",
```

Comment:
Maybe put this in alphabetical order up front. It doesn't make a difference to how the dependency list works, but I suspect `yarn`'ll auto sort this at some point and make someone else's PR messier.